R/functions__lazy.R

```diff
@@ -253,6 +253,70 @@ pl$last = function(column = NULL) {#-> Expr | Any:
 }
 
 
+#' Get the first `n` rows.
+#' @name pl_head
+#' @param column if dtype is:
+#' - Series: Take head value in `Series`
+#' - str or in: syntactic sugar for `pl.col(..).head()`
+#' @param n number of rows to take, NULL
+#' @keywords Expr_new
+#' @return Expr or head value of input Series
+#' @examples
+#' df = pl$DataFrame(
+#'   a = c(1, 8, 3),
+#'   b = c(4, 5, 2),
+#'   c = c("foo", "bar", "foo")
+#' )
+#' df$select(pl$head("a"))
+#'
+#' df$select(pl$head("a",2))
+#'
+#' pl$head(df$get_column("a"),2)
+pl$head = function(column = NULL, n = 10) {#-> Expr | Any:
```

Comment:
If `NULL` is not an acceptable input, then we may want to leave the argument without default and include an `is.missing()` check instead.

Reply:
oups good catch :)